src/web/routes/application/manual-address/manual-address.js

```diff
@@ -33,7 +33,7 @@ const contentSummary = (req) => ({
 })
 
 const isNavigable = (session) => {
```

Comment:
Suggestion (not PR blocking): as you are essentially composing Ramda functions this could be written as:
```
const isNavigable = compose(isNil, path(['claim', 'selectedAddress']))
```

Reply:
Done.